packages/components/src/StepperWizard/StepperWizard.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import { type Meta, type StoryObj } from '@storybook/react';
+import { useState } from 'react';
+
+import { Button, Grid } from '..';
+import { styled } from '../styles';
+
+import { StepperWizard } from './StepperWizard';
+import { type Steps } from './types';
+
+/**
+ * StepperWizard - фасад для работы с базовыми компонентами Stepper
+ *
+ * ### [Figma](https://www.figma.com/file/3ghN4WjSgkKx5rETR64jqh/Sirius-Design-System-(%D0%90%D0%9A%D0%A2%D0%A3%D0%90%D0%9B%D0%AC%D0%9D%D0%9E)?type=design&node-id=427-5578&mode=design&t=cAVE2DuNhtiUtoF6-0)
+ * ### [Guide]()
+ */
+const meta: Meta<typeof StepperWizard> = {
+  title: 'Components/StepperWizard',
+  component: StepperWizard,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof StepperWizard>;
+
+export const Interaction: Story = {
+  args: {
+    steps: [
+      { label: 'Step 1', error: false },
+      { label: 'Step 2', error: false },
+      { label: 'Step 3', error: false },
+    ],
+    activeStep: 1,
+  },
+  parameters: {
+    docs: {
+      disable: true,
+    },
+  },
+};
+
+const ButtonsWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+
+  padding: ${({ theme }) => theme.spacing(3, 10, 0, 10)};
+`;
+
+export const Example = () => {
+  const [activeStep, setActiveStep] = useState(0);
+
+  const steps: Steps = [
+    { label: 'Step 1' },
+    { label: 'Step 2' },
+    { label: 'Step 3', error: true },
```

Comment:
```suggestion
    { label: 'Step 3', isError: true },
```

1. По новому гайду должны иметь префикс.
2. Отображение ошибки можно оформить отдельной сторей